common/changes/@microsoft/rush/will-fix-last-install-flag_2022-09-13-21-16.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Autodetect if an autoinstaller's flag is outdated or node_modules folder is missing",
```

```suggestion
      "comment": "Fix an error that ocurred if an autoinstaller's  \"node_modules\" folder was manually deleted (GitHub #2987)",
```

Change messages should try to describe _what_ was solved rather than _how_ we solved it.  (See [this doc](https://rushjs.io/pages/best_practices/change_logs/) for some guidance.)